docs/user-guide/layer-settings.md

```diff
@@ -86,6 +90,56 @@ In particular, the user is allowed to:
 !!!note
     Cached tiles works with WMS only with standard gridsets (exponential). When MapStore uses custom scales, the tile cache can not be used. This because WMS TileGrid for OpenLayers (internal MapStore mapping library) and the TileGrid generated by GeoServer have different origins (top-left vs bottom-left). While for exponential resolutions the top left remains fixed (because there is only an integer number of tiles in the interval between top-left and bottom-left corner of the bbox, 1, 2, 4, 8 ....) with custom resolutions the number of tiles is not integer, so the coordinate of the tile of top-left may vary across scales. More details about this limitation in this [issue](https://github.com/geosolutions-it/MapStore2/issues/9025) on GitHub.
 
+## Fields
+
+From this section of the *Settings* panel, [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows the user to add aliases to layer fields.
+
+<img src="../img/layer-settings/fields_settings.jpg" class="ms-docimage"  style="max-width:500px;"/>
+
+The panel lists the fields that make up the layer. For each field the following is highlighted:
+
+* the **Name** of the field
+* the **Alias** of the field, which by default is empty
+* the **Type** of field
+
+The *Name* and the *Type* of the field cannot be modified, while the alias can be added for the user.
+
+Using the **Localize** <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button, a popup is opened through which the user can add the translation of the field according to the language.
+
+<img src="../img/layer-settings/localize-popup.jpg" class="ms-docimage"  style="max-width:500px;"/>
+
+In this case changing the language on [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) the title of the layer field is translated into the corresponding language.
+
+The layer field alias is defined instead of the original attribute name in the following [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) tools:
+
+* [Attribute Table](attributes-table.md)
+
+<img src="../img/layer-settings/lacalize-attribute-table.gif" class="ms-docimage"/>
+
+* [Filter layer](filtering-layers.md#layer-filter)
+
+<img src="../img/layer-settings/localize-filter.gif" class="ms-docimage"/>
+
+* [Identify](navigation-toolbar.md#identify-tool) (only `properties` output format)
+
+<img src="../img/layer-settings/localize-identify.gif" class="ms-docimage"/>
+
+* [Visual Style Editor](layer-settings.md#visual-editor-style)
+
+<img src="../img/layer-settings/localize-style.gif" class="ms-docimage"/>
+
+* [Charts Widget](widgets.md#chart) and [Table Widget](widgets.md#table)
+
+<img src="../img/layer-settings/localize-widget.gif" class="ms-docimage"/>
+
+Through the following toolbar, present in the *Fields* panel, the user can:
```

Comment:
Through the toolbar available on the top-center of the *Fields* panel, it is possible to: